processor/src/lib.rs

```diff
@@ -242,16 +244,22 @@ impl Process {
     fn execute_span_block(&mut self, block: &Span) -> Result<(), ExecutionError> {
         self.start_span_block(block)?;
 
+        let mut op_offset = 0;
+        let mut decorators = block.decorator_iter();
+
         // execute the first operation batch
-        self.execute_op_batch(&block.op_batches()[0])?;
+        self.execute_op_batch(&block.op_batches()[0], &mut decorators, op_offset)?;
+
```

Comment:
minor nit: let's get rid of this blank line.

Reply:
Done